Classes/Settings/SettingsViewController.swift

```diff
@@ -157,9 +158,19 @@ NewIssueTableViewControllerDelegate {
     }
 
     func onViewSource() {
-        guard let url = URL(string: Constants.URLs.repository)
-            else { fatalError("Should always create GitHub URL") }
-		presentSafari(url: url)
+        guard let client = client else {
+            ToastManager.showGenericError()
+            return
+        }
+
+        let repo = RepositoryDetails(
+            owner: "GitHawkApp",
+            name: "GitHawk",
```

Comment:
Should this and `"GitHawkApp" be added to the `Constants` file?
We use the name one other time in `onReportBug`: https://github.com/GitHawkApp/GitHawk/blob/751d4bc4e3974c66de24a31d611c151439b88e34/Classes/Settings/SettingsViewController.swift#L147